src/helpers.cpp

```diff
@@ -266,7 +267,7 @@ ada_really_inline size_t find_next_host_delimiter_special(
 }
 #elif ADA_SSE2
 ada_really_inline size_t find_next_host_delimiter_special(
-    std::string_view view, size_t location) noexcept {
+    std::string_view const view, size_t const location) noexcept {
```

Comment:
Why would you even pass a const size_t or a const std::string_view... it burdens the function signature and brings no benefit.

Reply:
Const there doesn't change the function signature, in fact, you shouldn't use const in the header files. But where the body of the function is, `const` helps to make the reader of the code (compiler or programmer) know that we're not gonna change this at all.